modules/renter/proto/renew.go

```diff
@@ -110,6 +111,7 @@ func (cs *ContractSet) Renew(oldContract *SafeContract, params ContractParams, t
 		// A revision mismatch might not be the host's fault.
 		if err != nil && !IsRevisionMismatch(err) {
 			hdb.IncrementFailedInteractions(contract.HostPublicKey())
+			err = errors.Extend(err, modules.ErrHostFault)
```

Comment:
same here, a small number of the errors after this deferred statement appear like they don't imply a problem on the host-side. Mostly AcceptTransactionSet